deploy/helm/block-explorer/templates/configmap-settings.yaml

```diff
@@ -21,45 +21,44 @@ data:
             "bech32PrefixValPub": "sifvaloperpub",
             "bech32PrefixConsAddr": "sifvalcons",
             "bech32PrefixConsPub": "sifvalconspub",
-            "bondDenom": "trwn",
+            "bondDenom": "rowan",
             "powerReduction": 1000000,
             "coins": [
                 {
-                    "denom": "trwn",
-                    "displayName": "TRWN",
-                    "displayNamePlural": "TRWNS",
+                    "denom": "rowan",
+                    "displayName": "ROWAN",
+                    "displayNamePlural": "ROWAN",
                     "fraction": 1000000
-                },           
+                },
                 {
                     "denom": "cusdt",
                     "displayName": "CUSDT",
                     "displayNamePlural": "CUSDTS",
                     "fraction": 1000000
-                },           
+                },
                 {
                     "denom": "cusdc",
                     "displayName": "CUSDC",
                     "displayNamePlural": "CUSDC",
                     "fraction": 1000000
-                },           
+                },
                 {
                     "denom": "clink",
                     "displayName": "CLINK",
                     "displayNamePlural": "CLINKS",
                     "fraction": 1000000
-                },           
+                },
                 {
                     "denom": "chot",
                     "displayName": "CHOT",
                     "displayNamePlural": "CHOTS",
                     "fraction": 1000000
                 }
-
             ],
             "gasPrice": 0.02,
             "coingeckoId": "sif"
         },
-        "genesisFile": "https://raw.githubusercontent.com/Sifchain/networks/feature/genesis/testnet/monkey-bars-testnet-3/genesis.json",
+        "genesisFile": "https://raw.githubusercontent.com/Sifchain/networks/feature/genesis/testnet/latest.json",
```

Comment:
For now this works. At some state we will need to parameterise this and the other options.